src/jobs/models/jobManager.js

```diff
@@ -65,17 +66,15 @@ module.exports.stopRun = async function (jobId, runId) {
 
 module.exports.deleteAllContainers = async function () {
     const configData = await configHandler.getConfig();
-    let runnerImage = configData.runner_docker_image;
     const jobConnector = require(`./${configData.job_platform.toLowerCase()}/jobConnector`);
-    let runnerImageWithoutTag = runnerImage.substring(0, runnerImage.indexOf(':'));
-    let result = await jobConnector.deleteAllContainers(runnerImageWithoutTag);
+    let result = await jobConnector.deleteAllContainers(PREDATOR_RUNNER_PREFIX);
```

Comment:
the function is receiving `PREDATOR_RUNNER_PREFIX` as the argument, but i'd expect it to receive `predator` or `JOB_PLATFORM_NAME`

In my opinion `PREDATOR_RUNNER_PREFIX` isn't really used (also in the other functions) and can be removed